packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/update/dedicated-server-monitoring-update.controller.js

```diff
@@ -29,76 +58,109 @@ angular.module('App').controller(
         });
     }
 
+    getMonitoringStatus() {
+      const { monitored, noIntervention } = this.server;
+      let monitoringStatus = 'disabled';
+
+      // proactive intervention
+      if (monitored && !noIntervention) {
+        monitoringStatus = 'proactive';
+      }
+
+      // no proactive intervention
+      if (monitored && noIntervention) {
+        monitoringStatus = 'no-proactive';
+      }
+
+      return monitoringStatus;
+    }
+
+    preselectOption() {
+      this.selectedOption = this.monitoringOptions.find(({ status }) => {
+        return status === this.getMonitoringStatus();
+      });
+    }
+
+    trackClick(hit) {
+      return this.atInternet.trackClick({
+        name: `${MONITORING_TRACKING_PREFIX}::${hit}`,
+        type: 'action',
+      });
+    }
+
+    trackConfirmClick(hit) {
+      return this.atInternet.trackClick({
+        name: `dedicated-server::monitoring-update-confirm::${hit}`,
+        type: 'action',
+      });
+    }
+
+    trackBannerMessages(statusHit, monitoringOptionHit) {
+      return this.atInternet.trackClick({
+        name: `dedicated-server::monitoring-update-banner::${statusHit}::${monitoringOptionHit}`,
+        type: 'action',
+      });
+    }
+
+    close() {
+      return this.$state.go('^');
+    }
+
     /**
      * Update monitoring.
      * @return {Promise}
      */
     update() {
-      const { monitored } = this.server;
+      this.trackClick('confirm');
+      this.trackConfirmClick(this.selectedOption.tracking);
```

Comment:
**question:** Do we need 2 trackings for the same action?

Reply:
Yes required from tracking team. Already validated.